fee/gas.go

```diff
@@ -10,6 +11,33 @@ const (
 	payloadGasSize = uint64(5)
 )
 
+func IsUseTxGas(version int32) bool {
```

Comment:
"is use" is not good english
it can be `fee.GasEnabled(version)` or `fee.UseGas(version)`

Reply:
it resolved in https://github.com/aergoio/aergo/pull/310/commits/cc4ffbf860757323fe485c62d925fa26dc0a1248